api/src/main/java/jakarta/persistence/Converter.java

```diff
@@ -22,38 +22,36 @@
 
 /**
  * Specifies that the annotated class is a converter and defines its
- * scope.  A converter class must be annotated with the <code>Converter</code>
+ * scope. A converter class must be annotated with the {@code Converter}
  * annotation or defined in the object/relational mapping descriptor as
  * a converter.
  *
- * <p>If the <code>autoApply</code> element is specified as
- * <code>true</code>, the persistence provider must automatically
- * apply the converter to all mapped attributes of the specified
- * target type for all entities in the persistence unit except for
- * attributes for which conversion is overridden by means of the
- * <code>Convert</code> annotation (or XML equivalent).
+ * <p>If {@link #autoApply autoApply = true}, the persistence provider
+ * must automatically apply the converter to all mapped attributes of
+ * the specified target type for all entities in the persistence unit
+ * except for attributes for which conversion is overridden by means
+ * of the {@link Convert} annotation (or XML equivalent).
  *
  * <p>In determining whether a converter is applicable to an attribute,
  * the provider must treat primitive types and wrapper types as
  * equivalent.
  *
- * <p>Note that Id attributes, version attributes, relationship
- * attributes, and attributes explicitly annotated as
- * <code>Enumerated</code> or <code>Temporal</code> (or designated as
- * such via XML) will not be converted.
+ * <p>Note that id attributes, version attributes, relationship
+ * attributes, and attributes explicitly annotated {@link Enumerated}
+ * or {@link Temporal} (or designated as such via XML) are not
+ * converted.
  *
- * <p>Note that if <code>autoApply</code> is <code>true</code>, the
- * <code>Convert</code> annotation may be used to override or disable
- * auto-apply conversion on a per-attribute basis.
+ * <p>Note that if {@code autoApply = true}, the {@link Convert}
+ * annotation may be used to override or disable auto-apply conversion
+ * on a per-attribute basis.
  *
- * <p>If <code>autoApply</code> is <code>false</code>, only those
- * attributes of the target type for which the <code>Convert</code>
- * annotation (or corresponding XML element) has been specified will
- * be converted.
+ * <p>If {@code autoApply = false}, only those attributes of the target
+ * type for which the {@link Convert} annotation (or corresponding XML
+ * element) has been specified will be converted.
```

Comment:
few lines above, _will not be converted_ was changed to _are not converted_, but here, _will be converted_ remains unchanged - I do not have strong opinion on which tense should be used, the only (sort of) requirement I have is to use the chosen one consistently

Reply:
OK, sorted. I got rid of lots of instances of "will be", which was very overused throughout the javadoc.